cobbler/settings/migrations/V3_0_0.py

```diff
@@ -223,5 +228,138 @@ def migrate(settings: dict) -> dict:
     for key in deleted_keys:
         helper.key_delete(key, settings)
 
-    # TODO: v2 to v3 script
+    # START: migrate-data-v2-to-v3
+    def serialize_item(collection, item):
+        """
+        Save a collection item to file system
+
+        :param collection: name
+        :param item: dictionary
+        """
+        filename = "/var/lib/cobbler/collections/%s/%s" % (collection, item["name"])
+
+        if settings.get("serializer_pretty_json", False):
+            sort_keys = True
+            indent = 4
+        else:
+            sort_keys = False
+            indent = None
+
+        filename += ".json"
+        with open(filename, "w+") as item_fd:
```

Comment:
Is there any specific reason to use `"w+"` mode, but not just `"w"`?

Reply:
Indeed there should be no difference as we only write to the file. I could indeed change it to `w`.